test/wallaby/experimental/chrome/logger_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule Wallaby.Experimental.Chrome.LoggerTest do
+  use ExUnit.Case, async: false
+
+  alias Wallaby.Experimental.Chrome.Logger
+  import ExUnit.CaptureIO
+
+  describe "parse_log/1" do
+    test "removes line numbers from the end of INFO logs" do
+      fun = fn ->
+        build_log(~s(http://localhost:52615//logs.html 13:14 "test"))
+        |> Logger.parse_log
+      end
+
+      assert capture_io(fun) == "test\n"
+    end
+
+    test "prints non-string data types" do
+      fun = fn ->
+        build_log(~s(http://localhost:52615//logs.html 13:14 1))
+        |> Logger.parse_log
+      end
+
+      assert capture_io(fun) == "1\n"
+
+      fun = fn ->
+        build_log(~s[http://localhost:52615//logs.html 13:14 Array(2)])
+        |> Logger.parse_log
+      end
+
+      assert capture_io(fun) == "Array(2)\n"
+    end
+
+    test "pretty prints json" do
+      message = ~s(http://localhost:54579//logs.html 13:14 "{\"href\":\"http://localhost:54579//logs.html\",\"ancestorOrigins\":{}}")
```

Comment:
I just submitted #301 to fix the issue that's giving you double slashes after the port. Once that's merged, would you mind rebasing and removing these double slashes?

Reply:
Yep, will do, I'm watching 301.